app/ts/components/requirements.tsx

```diff
@@ -1,230 +1,164 @@
-					<input type = 'checkbox' name = 'switch' class = 'check' checked = { requirementsMet } disabled = { true }/>
-					<p class = 'paragraph checkbox-text requirement'> { childDomainInfo.value.label } is wrapped </p>
-				</label>
-				<p class = 'paragraph dim'> { `Requires approve all from the owner (asked if needed)` } </p>
-				<SwitchAddress requirementsMet = { requirementsMet } account = { account } signingAddress = { correctSigningAddress }/>
-			</div>
-		</div>
-		<div class = 'grid-item' style = 'justify-self: end; justify-content: center;'>
-			<button class = 'button is-primary' { ...buttonDisabled ? { disabled: true } : {} } onClick = { button }>
-				Wrap { pending ? <Spinner/> : <></> }
-			</button>
+	<p class = 'subdomain-header'>{ checkBoxes.domainInfo.subDomain } </p>
+		<div class = 'grid-container'>
+			<Requirement checked = { checkBoxes.exists } primarytext = { `${ checkBoxes.domainInfo.subDomain } exists` } />
+			<Requirement checked = { checkBoxes.isWrapped } primarytext = { `${ checkBoxes.domainInfo.subDomain } is wrapped` } />
+			<Requirement checked = { checkBoxes.fusesBurned } primarytext = { `${ checkBoxes.domainInfo.subDomain } fuses are burnt` } secondaryText = { `The fuse "${ parentFuseToBurn }" is burnt` } />
 		</div>
 	</>
 }
 
-interface SetContentHashProps {
-	checkBoxes: Signal<CheckBoxes>
-	pendingCheckBoxes: Signal<CheckBoxes>
-	parentDomainInfo: Signal<DomainInfo>
-	childDomainInfo: Signal<DomainInfo> 
-	account: Signal<AccountAddress | undefined>
-	button: () => void
+interface CreateProps {
 	contentHashInput: Signal<string>
+	loadingInfos: Signal<boolean>
+	immutable: Signal<boolean>
 	handleContentHashInput: (input: string) => void
+	account: Signal<AccountAddress | undefined>
+	checkBoxes: OptionalSignal<CheckBoxes>
+	updateInfos: (showLoading: boolean) => Promise<void>
+	creating: Signal<boolean>
+	petalLockDeployed: Signal<boolean | undefined>
 }
 
-export const SetContentHash = ({ checkBoxes, parentDomainInfo, childDomainInfo, account, pendingCheckBoxes, button, contentHashInput, handleContentHashInput }: SetContentHashProps) => {
-	const pending = pendingCheckBoxes.value.childContentHashIsSet
-	const correctSigningAddress = getRightSigningAddress('setContentHash', childDomainInfo.value, parentDomainInfo.value)
-	const requirementsMet = checkBoxes.value.childContentHashIsSet
-	const buttonDisabled = account.value === undefined || !isSameAddress(account.value, correctSigningAddress) || requirementsMet || pending || !isValidContentHashString(contentHashInput.value)
+export const Create = ( { contentHashInput, loadingInfos, immutable, handleContentHashInput, account, checkBoxes, updateInfos, creating, petalLockDeployed }: CreateProps) => {
+	if (checkBoxes.deepValue === undefined) return <></>
+	const subDomain = checkBoxes.deepValue[checkBoxes.deepValue.length -1]?.domainInfo.subDomain
+	if (subDomain === undefined) throw new Error('missing subdomain')
+	// todo check that accesses are correct
```

Comment:
Don't forget!

Reply:
these are handled in the UI and error is thrown if the ownerships are not correct now